<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>411</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #1a1a1a;
color: #000000;</string>
   </property>
   <widget class="QLabel" name="heading">
    <property name="geometry">
     <rect>
      <x>-210</x>
      <y>0</y>
      <width>671</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Higher Jump</family>
      <pointsize>18</pointsize>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #d3d3d3;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;ENDE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="encryptButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #9caf88;
	color: #000000;
	font-size: 15px;
	font-weight: bold;
	border: none;
	border-radius: 25px;
	padding: 5px;

}
QPushButton::hover
{
	background-color: #5b7553;
}</string>
    </property>
    <property name="text">
     <string>ENCRYPT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decryptButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #ff7f50;
	color: #000000;
	font-size: 15px;
	font-weight: bold;
	border: none;
	border-radius: 25px;
	padding: 5px;

}
QPushButton::hover
{
	background-color: #ee4b2b;
}</string>
    </property>
    <property name="text">
     <string>DECRYPT</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>195</y>
      <width>87</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>algorithm :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>671</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/res/background.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>115</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Select File :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filepathEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>113</y>
      <width>341</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(165, 165, 165);
color: #000000;
font-weight: bold;
border: none;
border-radius: 2px;
padding: 3px</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>112</y>
      <width>65</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #ff3333;
	color: #ffffff;
	font-weight: bold;
	border: none;
	border-radius: 2px;
	padding: 5px;
}
QPushButton::disabled
{
	background-color: #5c5c5c;
}
QPushButton::pressed
{
	background-color: #ff4747;
}
</string>
    </property>
    <property name="text">
     <string>Browse ...</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rsaButton">
    <property name="geometry">
     <rect>
      <x>317</x>
      <y>170</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 79, 90);
</string>
    </property>
    <property name="text">
     <string>RSA</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="aesButton">
    <property name="geometry">
     <rect>
      <x>404</x>
      <y>170</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 79, 90);
</string>
    </property>
    <property name="text">
     <string>AES</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="blowfishButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 79, 90);
</string>
    </property>
    <property name="text">
     <string>Blowfish</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Enter the Password :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>238</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(165, 165, 165);
color: #000000;
font-weight: bold;
border: none;
border-radius: 2px;
padding: 3px</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="passwordcheckBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>275</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #ffffff;</string>
    </property>
    <property name="text">
     <string>Show Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>171</y>
      <width>233</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Select encryption / decryption</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>heading</zorder>
   <zorder>encryptButton</zorder>
   <zorder>decryptButton</zorder>
   <zorder>label</zorder>
   <zorder>filepathEdit</zorder>
   <zorder>browseButton</zorder>
   <zorder>label_2</zorder>
   <zorder>rsaButton</zorder>
   <zorder>aesButton</zorder>
   <zorder>blowfishButton</zorder>
   <zorder>label_3</zorder>
   <zorder>passwordEdit</zorder>
   <zorder>passwordcheckBox</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
